Content.Server/GameTicking/GameTicker.RoundFlow.cs

```diff
@@ -184,6 +185,16 @@ public void StartRound(bool force = false)
                 return true;
             });
 
+            var roleTimerManager = IoCManager.Resolve<RoleTimerManager>();
+
+            readyPlayers.RemoveAll(p =>
+            {
+                if (roleTimerManager.IsRoleTimeCachedYet(p.UserId))
+                    return false;
+                Logger.ErrorS("RoleTimers", $"Role timers for player {p} {p.UserId} have not been loaded yet.");
+                return true;
+            });
```

Comment:
I don't like the way this is handled, but at the same time I see some other code already started doing this so...

This should be handled the same way as player preferences: don't allow people to even ready up before database info is loaded.